### ISSUE TITLE
[BUG] Users unable to edit existing alt text on profile photos

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5571]
**Describe the bug**
Once a user has a profile with alt text added, they should be able to go in and edit that alt text without changing the photo. 

**To Reproduce**
_Using TalkBack,_ 
1. Go to 'Edit Profile info...'
2. Click on 'Edit Photo icon....'

**Expected behavior**
User is able to existing alt text and able to edit it and save changes. 
This is the screen I expected to be taken to when I selected Edit Photo Icon
![Image](https://github.com/user-attachments/assets/3f2aa4cb-7605-4731-a721-a1ddb82e9815)

**Actual behavior**
User is brought to the upload photo screen with "Camera" "Gallery" and "URL" options rather than the edit photo 

![Image](https://github.com/user-attachments/assets/16d5d682-a514-4cef-81c2-c3b50257c39b)

**Screenshots**

https://github.com/user-attachments/assets/52fa54db-ecb6-42cc-9aa0-8ab761212a01

Should work the same on all platforms/devices. 

**Additional context**

**Comment by acovher2** [comment]
https://github.com/user-attachments/assets/48bcc266-02b8-414f-bcbf-1ffda2230f00
@TodorBachvarov I've retested in 7.3.34 and this error keeps reoccurring.

**Comment by TodorBachvarov** [comment]
@acovher2, this should be fixed in version 7.3.38

**Comment by acovher2** [comment]
Verified as fixed in 7.3.41
